polaris.shopify.com/src/components/HomePage/HomePage.tsx

```diff
@@ -208,18 +212,30 @@ function HomePage({}: Props) {
                 />
               </div>
 
-              <div className={styles.Video}>
-                <video
-                  width="2250"
-                  height="1440"
-                  loop
-                  autoPlay
-                  muted
-                  playsInline
-                >
-                  <source src="/images/vscode.mp4" type="video/mp4" />
-                </video>
-              </div>
+              {useMotion ? (
+                <div className={styles.Video}>
+                  <video
+                    muted
+                    loop
+                    autoPlay
+                    playsInline
+                    width="2250"
+                    height="1440"
+                  >
+                    <source src="/images/vscode.mp4" type="video/mp4" />
+                  </video>
+                </div>
+              ) : (
+                <div className={styles.Poster}>
+                  <Image
+                    width="2250"
+                    height="1440"
+                    src={vsCodeImage}
+                    className={styles.Poster_Image}
```

Comment:
Nit: For classes, we use `ThisCasingWithoutUnderScores` in the rest of the project.